tools/gather-sysinfo/gather-sysinfo.go

```diff
@@ -174,6 +174,7 @@ func kniExpectedCloneContent() []string {
 		// KNI-specific CPU infos:
 		"/sys/devices/system/cpu/smt/active",
 		"/proc/sys/kernel/sched_domain/cpu*/domain*/flags",
+		//"/sys/devices/system/cpu/offline",
```

Comment:
why commented?

Reply:
I know but was failing every time. Just to be sure and then add must gather test.